modules/react/tooltip/lib/TooltipContainer.tsx

```diff
@@ -71,10 +84,10 @@ export const TooltipContainer = styled('div')<TooltipContainerProps>(
     },
 
     // offset tooltips by 2 pixels when a keyboard focus ring is detected
-    '[data-whatinput=keyboard] &': {
-      padding: `calc(${space.s} - 0.125rem)`,
-      ':before': {
-        margin: `calc(${space.xxxs} + 0.125rem)`,
+    'body:has(:focus-visible, .focus) &': {
+      padding: `calc(${system.space.x4} - ${calc.divide(system.space.x1, 2)})`,
```

Comment:
nit, you could use our `calc` functions here, but it is extra work

Reply:
Yeah updated from our conversation.